The hibernate-orm-5 module does not build with Hibernate 5.5.x or 5.6.x

When updating the Hibernate version in the Maven module `hibernate-orm-5` to any version of `5.5.x` or `5.6.x`, the `ORMStandaloneTestCase` fails with a NullPointerException. There is a breaking change between `5.4.x` and `5.5.x`, it looks like that starting with `5.5.x` the connection properties must be configured in `hibernate.properties`.

```
Running org.hibernate.bugs.JPAUnitTestCase
Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.195 sec
Running org.hibernate.bugs.ORMUnitTestCase
Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.02 sec <<< FAILURE!
org.hibernate.bugs.ORMUnitTestCase  Time elapsed: 0.019 sec  <<< ERROR!
java.lang.ExceptionInInitializerError
	at org.hibernate.testing.cleaner.DatabaseCleanerContext.<clinit>(DatabaseCleanerContext.java:17)
	at org.hibernate.testing.cleaner.DatabaseCleaner.clearSchemas(DatabaseCleaner.java:17)
	at org.hibernate.testing.junit4.BaseUnitTestCase.<clinit>(BaseUnitTestCase.java:39)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
	at org.junit.runners.BlockJUnit4ClassRunner.createTest(BlockJUnit4ClassRunner.java:250)
	at org.hibernate.testing.junit4.CustomRunner.getTestInstance(CustomRunner.java:139)
	at org.hibernate.testing.junit4.BeforeClassCallbackHandler.evaluate(BeforeClassCallbackHandler.java:25)
	at org.hibernate.testing.junit4.AfterClassCallbackHandler.evaluate(AfterClassCallbackHandler.java:25)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:252)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:141)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:112)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
Caused by: java.lang.IllegalArgumentException: java.lang.NullPointerException
	at org.hibernate.testing.cleaner.JdbcConnectionContext.<clinit>(JdbcConnectionContext.java:46)
	... 25 more
Caused by: java.lang.NullPointerException
	at java.base/java.util.concurrent.ConcurrentHashMap.putVal(ConcurrentHashMap.java:1011)
	at java.base/java.util.concurrent.ConcurrentHashMap.put(ConcurrentHashMap.java:1006)
	at java.base/java.util.Properties.put(Properties.java:1337)
	at org.hibernate.testing.cleaner.JdbcConnectionContext.<clinit>(JdbcConnectionContext.java:42)
	... 25 more
```

The test case can will succeed if you add `hibernate.connection.password` to `hibernate.properties` (No value for the property or any value)

--- a/orm/hibernate-orm-5/pom.xml
+++ b/orm/hibernate-orm-5/pom.xml
@@ -10,7 +10,7 @@
 	<properties>
 		<version.com.h2database>1.3.176</version.com.h2database>
 		<version.junit>4.13.2</version.junit>
-		<version.org.hibernate>5.4.24.Final</version.org.hibernate>
+		<version.org.hibernate>5.6.1.Final</version.org.hibernate>
 		<version.org.slf4j>1.7.25</version.org.slf4j>
 	</properties>
 

--- a/orm/hibernate-orm-5/src/test/resources/hibernate.properties
+++ b/orm/hibernate-orm-5/src/test/resources/hibernate.properties
@@ -10,6 +10,7 @@ hibernate.connection.driver_class org.h2.Driver
 #hibernate.connection.url jdbc:h2:mem:db1;DB_CLOSE_DELAY=-1;MVCC=TRUE
 hibernate.connection.url jdbc:h2:mem:db1;DB_CLOSE_DELAY=-1
 hibernate.connection.username sa
+hibernate.connection.password 
 
 hibernate.connection.pool_size 5
 

